src/js/pre-need/config/form.js

```diff
@@ -337,7 +338,7 @@ const formConfig = {
                       'view:hasServiceName': {
                         type: 'boolean'
                       },
-                      serviceName: _.omit('required', fullName),
+                      serviceName: _.omit(['required', 'properties.maiden'], fullName),
```

Comment:
Can we do this omit call once and reuse the result? Also, it seems like we're mostly removing maiden name, I wonder if it would make more sense to remove it from fullName in the schema and only add it in the config in the place(s) we need it?

Reply:
Sounds good @jbalboni, updated.